projects/laji/src/app/shared-modules/datatable/datatable/datatable.component.ts

```diff
@@ -287,12 +287,43 @@ export class DatatableComponent implements AfterViewInit, OnInit, OnChanges, OnD
   }
 
   onSort(event) {
-    this.sorts = event.sorts;
-    this.sortRows(event.sorts);
+    this.sorts = this.updateSorts(event);
+    this.sortRows(this.sorts);
     this.changeDetectorRef.detectChanges();
+    event.sorts = this.sorts;
```

Comment:
Is this necessary? Altering function parameters in mutable fashion is shady business.

Reply:
Actually not, I should've put a new object into event emitter that is the new sort and rest from destructuring the event, I'll fix it. And I'll try to figure out what's going on with that bug.

Reply:
It's fixed now, I had set wrong column property as sort prop, which worked most of the time by accident but not for taxon.